llvm-runtimes/compiler-rt-sanitizers/compiler-rt-sanitizers-22.0.0.9999.ebuild

```
@@ -117,6 +120,17 @@ src_configure() {
 		local -x CC=${CHOST}-clang
 		local -x CXX=${CHOST}-clang++
```

```suggestion
		local -x CC=${CHOST}-clang-${SLOT}
		local -x CXX=${CHOST}-clang++-${SLOT}
```

I guess. Otherwise, you may end up mixing configs from a different version.